x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -29,6 +29,7 @@ export const EPM_API_ROUTES = {
   DELETE_PATTERN: EPM_PACKAGES_ONE,
   FILEPATH_PATTERN: `${EPM_PACKAGES_FILE}/{filePath*}`,
   CATEGORIES_PATTERN: `${EPM_API_ROOT}/categories`,
+  STATS_PATTERN: `${EPM_PACKAGES_MANY}/{pkgName}/stats`,
```

Comment:
This seems to be called summary elsewhere. Perhaps we should make the var names consistent, to mentally correlate them a bit better. 

If the route already exists and is `/stats`, I can see giving the route's var name the same as the route itself. But then it's switched to called summary on the layer right outside that. Consider naming the pattern var name summary as well. Reduces the mental context switch to just here, next to the route, which is a little more concrete to reason about.

Reply:
I have renamed everything else `*Stats*` in order keep consistent with this decision here 😄 .  Thanks for highlighting that - _"me culpa"_ 😞